<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>848</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>B-树演示</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeView" name="mainTreeView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>401</width>
      <height>681</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAdd">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>790</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>添加键值</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineKeyword">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>759</y>
      <width>401</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>输入一个整数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRemove">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>790</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除键值</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonExpand">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>720</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>全部展开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCollapse">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>720</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>收起到根节点</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonQuery">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>790</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查找键值</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonReset">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>790</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空树</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>810</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>清空树时将会将树的阶设为输入的整数</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>725</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>当前 B-树的阶：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelOrder">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>730</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
